content/workload-reduction-toolkit/evaluate-workload-measures.html.erb

```diff
@@ -64,30 +64,29 @@ colour: green
 
             <%= render('/partials/resource_card.*',
               title: "Evaluate progress with staff",
-              href: "TBC",
+              href: "#{@base_url}/workload-reduction-toolkit/evaluate-workload-measures/evaluate-progress-with-staff",
               body: "Show staff the actions you took to address workload issues and gather feedback.",
               tag: "Template") %>
 
             <%= render('/partials/resource_card.*',
               title: "Have structured conversations with staff to evaluate outcomes",
-              href: "TBC",
+              href: "#{@base_url}/workload-reduction-toolkit/evaluate-workload-measures/structured-conversations-with-staff",
               body: "Talk to staff and teachers about their biggest workload issues.",
               tag: "Template") %>
 
             <%= render('/partials/resource_card.*',
               title: "Update your workload action plan",
-              href: "TBC",
+              href: "#{@base_url}/workload-reduction-toolkit/evaluate-workload-measures/update-your-workload-action-plan",
               body: "Evaluate the actions you took to reduce workload in your school.",
-              tag: "Template",
-              details: { reading_time: "1 minute", created_by: "Notre Dame High School"}) %>
+              tag: "Template") %>
```

Comment:
This should be an 'Example' rather than a template.

It also still needs the reading time and created by details, but the school name should be Notre Dame High School (Secondary)